### PR TITLE
Brackets cheat sheet added

### PR DESCRIPTION
**Pull request by f4yis**
- [ ] New Instant Answer
  - [x] Cheat Sheets: **`New Brackets Cheat Sheet`**

Instant Answer Page: https://duck.co/ia/view/brackets_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@f4yisThanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Brackets Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/brackets_cheat_sheet_mobile.png?BuTa1KA8Vn)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Brackets Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/brackets_cheat_sheet_mobile.png?MqSHTO5u02)

File: share/goodie/cheat_sheets/json/brackets.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
+{
+	"id": "brackets_cheat_sheet",
+	"name": "Brackets text editor",
+	"description": "Keyboard shortcuts for Brackets text editor",
+	"metadata": {
+		"sourceName": "Github Brackets repository",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/brackets.json]: "Github" would be enough here

File: share/goodie/cheat_sheets/json/brackets.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+	],
+	"sections": {
+		"General": [{
+			    "val": "Increase Font Size",
+			    "key": "[Ctrl] [+]"
+		}, {
+			    "val": "Decrease Font Size",
+			    "key": "[Ctrl] [-]"
+		}, {
+			    "val": "BToggle Line Comment",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/brackets.json]: is it BToggle or Toggle?

File: share/goodie/cheat_sheets/json/brackets.json (hunk 1 above)
Comment (f4yis) [inline share/goodie/cheat_sheets/json/brackets.json]: Thanks

File: share/goodie/cheat_sheets/json/brackets.json (hunk 2 above)
Comment (f4yis) [inline share/goodie/cheat_sheets/json/brackets.json]: It was a typo. Thanks

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Brackets Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/brackets_cheat_sheet_mobile.png?o292SO26sb)

File: share/goodie/cheat_sheets/json/brackets.json [hunk 3]
```diff
@@ -0,0 +1,176 @@
+{
+	"id": "brackets_cheat_sheet",
+	"name": "Brackets text editor",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/brackets.json]: I think "Brackets" would be enough here

File: share/goodie/cheat_sheets/json/brackets.json [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+		"brackets for linux",
+		"linux brackets",
+		"windows brackets"
+	],
+	"template_type": "keyboard",
+	"section_order": [
+		"General"
+	],
+	"sections": {
+		"General": [{
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/brackets.json]: Lets split this up into smaller sections.

**Comment by pjhampton** [comment]
This PR looks as if it has gone stale. Closing

**Pull request closed without merging**